src/1215_a_cpp_approach_to_physical_units.bs

```diff
@@ -205,21 +221,27 @@ ISO 80000-1:2009(E) Quantities and units - Part 1: General
     which are quantities, is also considered to be a quantity.
 - The concept ’quantity’ may be generically divided into, e.g. ‘physical quantity’,
     ‘chemical quantity’, and ‘biological quantity’, or ‘base quantity’ and ‘derived quantity’.
+- Examples of quantities are: mass, length, density, magnetic field strength, etc.
 
 <dfn lt="kind of quantity|kind">kind of quantity, kind</dfn>
 
 - Aspect common to mutually comparable [=quantities=].
-- The division of the concept ‘quantity’ into several kinds is to some extent arbitrary
-    - i.e. the quantities diameter, circumference, and wavelength are generally considered
-        to be [=quantities=] of the same kind, namely, of the kind of quantity called length.)
+- The division of the concept ‘quantity’ into several kinds is to some extent arbitrary. For 
+    example, the quantities diameter, circumference, and wavelength are generally considered
+    to be [=quantities=] of the same kind, namely, length.
 - [=Quantities=] of the same kind within a given [=system of quantities=] have the same quantity
     [=dimension=]. However, [=quantities=] of the same [=dimension=] are not necessarily of
-    the same kind.
+    the same kind. For example, the absorbed dose and the dose equivalent have the same
+    dimension. However, the former measures the absolute amount of radiation one receives
+    wherase the latter is a weighted measurement taking into account the kind of radiation on 
+    was exposed to.
```

Comment:
Those definitions are taken straight from ISO 80000 and I wouldn't change or amend them in any form.

Reply:
Okay. I didn't check and assumed that you made slight adjustments. My bad!

Reply:
Nevertheless, I would add examples at the bottom. For people new with the topic, it is already hard enough to understand.